### PR TITLE
Update CONTRIBUTORS.md and CONTRIBUTING.md with calls for translations

### PR DESCRIPTION
**Pull request by robertsky**
## Description
Update CONTRIBUTORS.md and CONTRIBUTING.md with calls for translations

## How Has This Been Tested?
Edited and viewed the markdown files on github.

## Types of changes
Update CONTRIBUTING.md and CONTRIBUTORS.md.

## Checklist:

- [x] No code changed.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -69,3 +69,13 @@ To add a new documentation page, you'll have to create a Markdown file in the [d
 ## Reporting Security Issues
 
 Please see [SECURITY.md](./SECURITY.md).
+
+## Internationalizing Gutenberg Plugin
```
Comment (aduth) [inline CONTRIBUTING.md]: As defined, I might think "Localizing" would be the preferred term here for contributors looking to submit strings.

https://www.w3.org/International/questions/qa-i18n

**Review by aduth — APPROVED** [review]
Thanks for adding this section. Left one minor note about choice of terminology. Let me know how you'd like to proceed.

File: CONTRIBUTING.md (hunk 1 above)
Comment (robertsky) [reply CONTRIBUTING.md]: Okie. I have updated the heading. :)

**Comment by robertsky** [comment]
@aduth I have updated the changes as you have suggested. :)

**Comment by aduth** [comment]
Thanks!

**Pull request merged**